package.json

```diff
@@ -5,16 +5,15 @@
     "production": "cross-env NODE_ENV=production postcss public/tailwind.css -o public/build/tailwind.css",
     "start": "npm run serve"
   },
-  "dependencies": {
-    "autoprefixer": "^9.5.1",
-    "tailwindcss": "^1.0"
-  },
+  "dependencies": {},
   "devDependencies": {
     "@fullhuman/postcss-purgecss": "^1.2.0",
```

Comment:
There is no need to include purgecss anymore as it comes bundled with tailwind already :)